mlflow/utils/databricks_utils.py

```diff
@@ -1,10 +1,12 @@
 import os
 import logging
 import subprocess
+import mlflow._spark_autologging
```

Comment:
Could we actually add a file like `mlflow/_spark_utils.py` containing common utils for working with Spark and move the `get_active_spark_session` into that file? We could then import and use the `get_active_spark_session` method here and in `mlflow/_spark_autologging.py` as follows:


```suggestion
from mlflow._spark_utils import get_active_spark_session
```

Reply:
I think this is better than brutally importing the mlflow._spark_autologging. Will make these functions more organized. Will do!